src/pages/CreatePool/StaticFeeSelector.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useRef, useEffect } from 'react'
+import styled from 'styled-components'
+import { useMedia } from 'react-use'
+const ToggleButton = styled.span<{ size?: string; element?: HTMLSpanElement; firstRender?: boolean }>`
```

Comment:
`ToggleButton` `ToggleElement` `ToggleWrapper`... seem duplicate with FeeTypeSelector

Reply:
yeah, they are toggle button elements with some different CSS. But I think it's better to split them out to different components than write a complicated component to reuse.